src/components/Navbar/index.tsx

```diff
@@ -46,7 +46,7 @@ const DashboardButton = (props?: ButtonProps) => {
   return (
     <Button
       as={ReactRouterLink}
-      to={isAuthenticated ? Routes.dashboard.base : Routes.auth.signIn}
+      to={isAuthenticated ? Routes.dashboard.processes : Routes.auth.signIn}
```

Comment:
`Routes.dashboard.processes` needs to be used with `generatePath`, since it has conditional params.